### DISCUSSION TITLE
Is that possible to create dynamic pages, like passing parameter in the url

### DISCUSSION THREAD

**Discussion by CWSPS154** [Z3d0X/filament-fabricator#183] [Q&A]
I have a scenario to create a gallery page, that I created using fabricator blocks and pages, there is button to view more. I need to show that data in a separate page. For that I need to pass id along with URL and take the data. Is this possible

**Comment by dave2084** [comment]
I'm interested in this too, I'd like to and a blog model where we have two pages, one for the archive which would a list of posts and another where I can hook into a single block post passing an ID or slug of the blog model instance to be displayed.

**Reply by sasaorasanin** [reply]
Easy solution would be to make your blog component use like `request()->query('post')`, and just to generate new page with uri `/post`, then when you generate link you can do `/post?post=1`.

If you want clean url like `/post/my-first-post` I guess you would need to make uri `/post/{post}` and then to handle finding that page with your custom logic.
